pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -62,6 +62,12 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		})
 	}
 
+	updateToMasqueradeDeafultNetwork := func() {
```

Comment:
Default

Reply:
Hi @booxter . Thank you for the notes. I made some updates in the PR, what do you think now?